### ISSUE TITLE
'product-search-pricing-flow-mobile' should be able to search with price - e2e fails

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#2642]
product-search-pricing-flow-mobile.e2e.spec.ts - should be able to search with price

![image](https://user-images.githubusercontent.com/11735817/58253978-0b3c0900-7d6a-11e9-851a-2c6c10f4c647.png)

**Comment by Zeyber** [comment]
Blocked by #2135

**Comment by plabadie** [comment]
I suggest we close this ticket as "Won't fix"

* I did a smoke test against dev17 and tests are still failing.
* This test is already fixed by GH-2680
